client/src/app/routes/stardust/Foundry.tsx

```diff
@@ -161,57 +168,67 @@ const Foundry: React.FC<RouteComponentProps<FoundryProps>> = (
                         </div>
                     </div>
                 </div>
-                <div className="section">
-                    <div className="section--header row row--tablet-responsive middle space-between">
-                        <div className="row middle">
-                            <h2>Token Info</h2>
-                        </div>
-                    </div>
-                    <div className="section--data">
-                        <div className="label">
-                            Token scheme
-                        </div>
-                        <div className="value code row middle">
-                            <span className="margin-r-t">
-                                {tokenScheme}
-                            </span>
-                        </div>
-                    </div>
-                    <div className="section--data">
-                        <div className="label">
-                            Maximum supply
-                        </div>
-                        <div className="value code row middle">
-                            <span className="margin-r-t">
-                                {maximumSupply}
-                            </span>
+                <TabbedSection
+                    tabsEnum={FOUNDRY_PAGE_TABS}
+                    tabOptions={{
+                        [FOUNDRY_PAGE_TABS.Features]: {
+                            disabled: optional(foundryOutput.features).isEmpty()
+                        },
+                        [FOUNDRY_PAGE_TABS.ImmutableFeature]: {
+                            disabled: optional(foundryOutput.immutableFeatures).isEmpty()
+                        }
+                    }}
+                >
+                    <div className="section">
+                        <div className="section--header row row--tablet-responsive middle space-between">
+                            <div className="row middle">
+                                <h2>Token Info</h2>
```

Comment:
Remove the Token info title because the title is shown on the Tab.
![image](https://user-images.githubusercontent.com/52735957/222459786-47c681c2-1ae8-43cd-9d98-c96d5b0d3763.png)
Also the same for Features and Immutable Features sections.